tests/Unit/EnumTest.php

```diff
@@ -64,4 +66,16 @@ public function testEnumSerialization()
             $eventStatus::memberVal
         );
     }
+
+    public function testPaymentMethodEnum()
```

Comment:
It would be nice to also do a non-happy path test here, to check that it's still rejecting invalid enums

Reply:
Can you define what an invalid enum would be? And is it always the same for all enums?

Reply:
For example, if the value was `"dummy"`, it would not be considered valid